website/content/en/status/report-2024-10-2024-12/pinephone.adoc

```diff
@@ -0,0 +1,15 @@
+=== Pinephone Pro Support
+Links: +
+link:https://codeberg.org/Honeyguide/freebsd-pinephonepro[Repository on Codeberg] URL: link:https://codeberg.org/Honeyguide/freebsd-pinephonepro[]
+
+Contact: Toby Kurien <toby@tobykurien.com>
+
+The project to port FreeBSD over to the Pinephone Pro is progressing. The aim of this project is to step by step support components of the Pinephone Pro in FreeBSD so that the device one day might be usable as a highly mobile FreeBSD device.
+
+In this quarter, a driver for the RK818 power management IC was implemented, enabling the device regulators. A driver for the real-time clock was also implemented, allowing the system to keep time between reboots. A driver for the RK818 battery charger and battery monitor was written to allow the battery to be charged via USB, and to retrieve some battery information like voltage and charging status via sysctl. The code repository has been updated with scripts and documentation on how to compile the custom kernel and device tree, and patch a FreeBSD 15-CURRENT image with them so that it boots on the Pinephone Pro.
```

Comment:
```suggestion
In this quarter:

* A driver for the RK818 power management IC was implemented, enabling the device regulators.
* A driver for the real-time clock was also implemented, allowing the system to keep time between reboots.
* A driver for the RK818 battery charger and battery monitor was written to allow the battery to be charged via USB, and to retrieve some battery information like voltage and charging status via sysctl.
* The code repository has been updated with scripts and documentation on how to compile the custom kernel and device tree, and patch a FreeBSD 15-CURRENT image with them so that it boots on the Pinephone Pro.
```